.github/workflows/linux.yml

```diff
@@ -62,6 +62,42 @@ jobs:
           path:
             opensc*.tar.gz
 
+  build-arm64:
+    name: build-${{ matrix.dist }}-${{ matrix.arch }}
+    strategy:
+      matrix:
+        dist: ['22.04', '24.04']
+        arch: ['arm64']
+    runs-on: ubuntu-${{ matrix.dist }}${{ matrix.arch == 'arm64' && '-arm' || '' }}
+    steps:
+      - uses: actions/checkout@v4
+      - uses: awalsh128/cache-apt-pkgs-action@v1.4.3
+        with:
+          packages: ${{ env.BASE_DEPS }}
+          version: apt-${{ matrix.dist }}-${{ matrix.arch }}-base
+          execute_install_scripts: true
+      - run: .github/setup-linux.sh
+      - run: .github/build.sh dist
+      - name: Upload test logs
+        uses: actions/upload-artifact@v4
+        if: failure()
+        with:
+          name: ubuntu-${{ matrix.dist }}-${{ matrix.arch }}-test-logs
+          path: |
+            tests/*.log
+            src/tests/unittests/*.log
+      - uses: actions/cache@v4
+        id: cache-build
+        with:
+          path: ./*
+          key: ${{ runner.os }}-${{ matrix.dist }}-${{ matrix.arch }}-${{ github.sha }}
```

Comment:
Do we need this step? In existing workflows it is used to cache the build between the stages (tests after build) and I do not think we have here any state following up

Reply:
I was thinking maybe make this thing more general. Maybe unify some other builds also eg: x64.
I see lot of duplication already there.
I did not figure out how to depend specifically on one matrix combination. Maybe it is ok to wait until whole matrix ends?

Reply:
I think so. As long as they will run similar time, it should be ok to wait for all of them.

Reply:
Done